apps/mediaviewer/src/Mediaviewer.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div id="mediaviewer" class="uk-background-secondary">
+    <button @click="activeItem++">Next</button>
+    <div class="uk-position-center">
+      <transition name="custom-classes-transition" enter-active-class="uk-animation-slide-right-small" leave-active-class="uk-animation-slide-left-medium uk-animation-reverse">
+        <img v-show="!loading" ref="mediaviewerMedia" :src="image" alt="" style="max-width:90vw;max-height:90vh">
+      </transition>
+    </div>
+    <oc-spinner class="uk-position-center" v-if="loading" size="large" />
+  </div>
+</template>
+<script>
+import _ from 'lodash'
+import { mapGetters } from 'vuex'
+import imageSrc from '../../../src/mixins/imageSrc.js'
```

Comment:
ouch - this is bad .... and should not be necessary - the directive is already registered in Vue

Reply:
In this case, it's a mixin. Not using the directive here as I needed the promise based attempt.

Reply:
never the less - importing a file from outsode of the app can cause major issues in transpiling - been there - done that

Reply:
better use a plugin I guess - https://vuejs.org/v2/guide/plugins.html